### DISCUSSION TITLE
--keep-going in manifest mode should display which packages failed

### DISCUSSION THREAD

**Discussion by ras0219-msft** [microsoft/vcpkg#26708] [Ideas]
After running `vcpkg install --keep-going`, if packages failed, vcpkg should display specifically which ones. We don't want to print a summary in the no-op case since it's called often as part of user's configure steps.

**Classic mode, today**
```
$ .\vcpkg install zlib rapidjson
/* package builds */
Total elapsed time: 16.94 s

RESULTS
    rapidjson:x86-windows: SUCCEEDED: 7.778 s
    zlib:x86-windows: BUILD_FAILED: 196.3 ms

SUMMARY FOR x86-windows
    SUCCEEDED: 1
    BUILD_FAILED: 1

/* usage information */
```
**Manifest mode, today**
```
$ .\vcpkg install zlib rapidjson
/* package builds */
Total elapsed time: 15.79 s
/* usage information */
```
**Suggested manifest mode**
```
$ .\vcpkg install zlib rapidjson
/* package builds */
Total elapsed time: 15.79 s

RESULTS
    zlib:x86-windows: BUILD_FAILED: 196.3 ms

/* usage information */
```

**Comment by Osyotr** [comment]
It should also exit with error if some packages failed. Right now cmake will continue configuration even when some packages were not installed.
